### PR TITLE
Fix #213 trash does not appear on kindergarten

### LINKED ISSUE #213: El tacho para eliminar bloques aparece y desaparece
En el espacio para trabajar con los bloques a veces aparece el tacho de basura para eliminar bloques y otras veces no. 
Suele **no** aparecer apenas se carga el ejercicio y aparece cuando:

* se hace click en el botón de ejecutar o
* se abre, por ejemplo, el devtools de chrome.

Dejo un video:

https://drive.google.com/drive/folders/1U_xruwGEHm8ghr2VTATj8ETy4ODVNlj4

### PR DESCRIPTION
**Pull request by flbulgarelli**
# :dart: Goal

To fix trash relocation errors

# :memo: Details

The problem was that a call to `_relocateTrash` after initializing the workspace was missing

# :camera_flash: Screenshots

## Kindergarten

image
Screenshot_2020-11-11_15-13-55

## Primary

image
Screenshot_2020-11-11_15-12-49

## Classroom

image

### CODE REVIEW

**Review by luchotc — APPROVED** [review]
Great! :clap: 
I'll wait for @faloi approval and merge it :smile:

File: lib/render/editor/editor.html.erb
Comment (faloi) [inline lib/render/editor/editor.html.erb:235]: This will be a problem if, for any reason, `readOnly` constant turns into a variable in the future. Perhaps we can always subscribe to the event, the actual relocation won't happen while the editor is read only.

**Comment by luchotc** [comment]
I'll merge it as we need this fix but leave @faloi comment as an issue!

**Pull request merged**
